<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="2" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="24" started-at="2022-03-31T04:20:38Z" finished-at="2022-03-31T04:20:38Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="24" started-at="2022-03-31T04:20:38Z" finished-at="2022-03-31T04:20:38Z">
      <class name="org.com.Demo">
        <test-method status="PASS" signature="tc1()[pri:0, instance:org.com.Demo@e580929]" name="tc1" duration-ms="9" started-at="2022-03-31T09:50:38Z" finished-at="2022-03-31T09:50:38Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tc1 -->
        <test-method status="FAIL" signature="tc2()[pri:0, instance:org.com.Demo@e580929]" name="tc2" duration-ms="0" started-at="2022-03-31T09:50:38Z" finished-at="2022-03-31T09:50:38Z">
          <exception class="java.lang.Error">
            <message>
              <![CDATA[Unresolved compilation problems: 
	SoftAssert cannot be resolved to a type
	SoftAssert cannot be resolved to a type
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Error: Unresolved compilation problems: 
	SoftAssert cannot be resolved to a type
	SoftAssert cannot be resolved to a type

	at org.com.Demo.tc2(Demo.java:14)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:715)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:907)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1237)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:51)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:85)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1197)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1122)
	at org.testng.TestNG.run(TestNG.java:1030)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:207)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:178)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Error -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tc2 -->
      </class> <!-- org.com.Demo -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
